domain/build.gradle.kts

```
@@ -0,0 +1,9 @@
+plugins {
+    id("java-library")
+    id("org.jetbrains.kotlin.jvm")
```

순수 코틀린 모듈이라면 아래처럼 사용할수 있을거 같아요 😄 
```suggestion
    id("kotlin")
```